helmfile/values/networkpolicies/service/harbor.yaml.gotmpl

```diff
@@ -140,7 +158,11 @@ policies:
       egress:
         - rule: egress-rule-dns
         - rule: egress-rule-ingress
+        {{- if eq .Values.harbor.persistence.type "s3" }}
         - rule: egress-rule-object-storage
```

Comment:
I guess this can be done in another issue, but it would be nice to have this be named `egress-rule-s3-object-storage` now that we have swift as well